src/main/java/org/traccar/forward/PositionForwarderMqtt.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2022 - 2023 Anton Tananaev (anton@traccar.org)
```

Comment:
Here as well

Reply:
" * Copyright 2023 - 2024 Anton Tananaev (anton@traccar.org) " is this correct ? or will this be 24-25

Reply:
It should be just 2024